### ISSUE TITLE
Screen ID Component - Update contrast 

### ISSUE THREAD

**Issue by bignamehere** [USDA-FSA/fsa-style#220]
The grey background is currently set to a contrast ratio of 3.6:1 and needs to be a 4.5:1. Recommendation to utilize the "$color-fsa-tertiary" for the grey background bar, as this will mirror the Disclaimer component above the App Header.

**Also, look into adding an Aria-label as per 508 team recommendation.**

Current:
![image](https://user-images.githubusercontent.com/2124049/37410119-d5d44f1e-276d-11e8-81c5-423f7dda5231.png)

Recommended:
![image](https://user-images.githubusercontent.com/2124049/37410082-c183c3d2-276d-11e8-87ad-8fb1f80bbdc4.png)

**Comment by francisrupert** [comment]
Meh, I actually did that on purpose knowing full-well it failed contrast ratio, with a general thought of "this is unique enough a feature to accept since I don't care what it looks like."

Thoughts?

**Comment by bignamehere** [comment]
Yeah, the issues is more than Screen ID is supposed to be a requirement for every page in every FSA application. It have very little usage in the field, from what I am told, but it wouldn't hurt to make sure it is compliant.
